docs/modernization/WebViewCompatible.md

```diff
@@ -0,0 +1,224 @@
+---
+title: WebViewCompatible control for Windows Forms and WPF
+author: granitestatehacker
+description: The Windows Community Toolkit provides a version of the UWP web view control that can be used in WPF and Windows Forms applications. This control embeds a view into your application that renders web content in one of two ways. For client environments that support the WebViewControl (Windows 10), that implementation is used. For legacy systems, System.Windows.Controls.WebBrowser implements the view.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, WebView, Windows Forms, WPF
+---
+
+# WebViewCompatible controls for Windows Forms and WPF applications
+
+The **WebViewCompatible** controls show web content in your Windows Forms or WPF desktop application.
+
+![Web View Samples](../resources/images/Controls/WebView/web-view-samples.png)
+
+These controls use the one of two rendering engines.  For Windows 10 client environments, the newer Microsoft Edge rendering engine is used to embed a view that renders richly formatted HTML content from a remote web server, dynamically generated code, or content files.  For legacy systems, the System.Windows.Controls.WebBrowser is used, providing Internet Explorer engine-based rendering.
+
+## About WebViewCompatible controls
+
+The Windows Forms version of this control is coming soon. It will be located in the **Microsoft.Toolkit.Win32.UI.Controls.WinForms** namespace. 
+
+The WPF version is located in the **Microsoft.Toolkit.Win32.UI.Controls.WPF** namespace. 
+
+You can find additional related types (such as event args classes) in the **Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT** namespace.
+
+Internally, these controls wrap the [WebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol) class, so they provide a subset of members from that class. The [WebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol) is similar to the [WebView](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.webview) class, but it is designed to run out of process in a desktop application (such as a WPF or Windows Forms application) and it supports a smaller set of members.  Because WebViewCompatible wraps [WebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol) and WebBrowser, it provides a simpler subset of common functionality.
```

Comment:
>provide a subset of members from that class [](start = 142, length = 43)

This is an important statement: there is a subset of the Win32 WebView and WebBrowser APIs that are provided